Makefile

```diff
@@ -136,8 +146,45 @@ test-unit:
 # Build and vendor tarets
 #---------------------------------------------------------
 
-# Vendor target: "go mod vendor" resets the main module's vendor directory to include all packages needed to build and 
-# test all of the module's packages based on the state of the go.mod files and Go source code.
+.PHONY: build
+## Build: using operator-sdk to build a new image
+build:
```

Comment:
@otaviof , Could you please fix the duplication here?
there's already a `build` target which does a typical `go build`. 

Sync-ing this with master shows up the conflict.

We need to have both targets
- `make build` : Compiles and does a simple `go build` as we have in `master`
-  and another target which does a lot more than that with operator-sdk

Reply:
Sure! It's now having both targets, `build` and `build-image`, please consider.